web/views/icons/stylus.tmpl

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
+  <path d="M 2.8691406 0 C 1.8391417 0 1 0.83914165 1 1.8691406 L 1 7.1191406 L 0 7.1191406 L 0 9 L 1 9 L 1 14.130859 C 1 15.160858 1.8391417 16 2.8691406 16 L 13.130859 16 C 14.160858 16 15 15.160858 15 14.130859 L 15 9 L 16 9 L 16 7.1191406 L 15 7.1191406 L 15 1.8691406 C 15 0.83914165 14.160858 0 13.130859 0 L 2.8691406 0 z M 3.3007812 0.61914062 L 12.699219 0.61914062 C 13.549218 0.61914063 14.25 1.3196884 14.25 2.1796875 L 14.25 7.1191406 C 14.25 7.1191406 14.310001 7.6894531 15.25 7.6894531 L 15.25 8.4394531 C 14.310001 8.4394531 14.25 9 14.25 9 L 14.25 13.820312 C 14.25 14.680312 13.549218 15.380859 12.699219 15.380859 L 3.3007812 15.380859 C 2.4507821 15.380859 1.75 14.680312 1.75 13.820312 L 1.75 9 C 1.75 9 1.6899991 8.4394531 0.75 8.4394531 L 0.75 7.6894531 C 1.6899991 7.6894531 1.75 7.1191406 1.75 7.1191406 L 1.75 2.1796875 C 1.75 1.3196884 2.4507821 0.61914062 3.3007812 0.61914062 z M 8.5605469 1.9804688 C 5.8105496 1.9804688 4.1308594 3.4902364 4.1308594 5.4902344 C 4.1308594 7.1802327 5.4193769 8.2592194 7.359375 8.9492188 C 8.7593736 9.4392183 9.3203125 9.8600788 9.3203125 10.580078 C 9.3203125 11.350077 8.6809363 11.849609 7.4609375 11.849609 C 6.3409386 11.849609 5.2292962 11.48914 4.5292969 11.119141 L 3.9902344 13.289062 A 7.7 7.7 0 0 0 7.3105469 14.019531 C 10.510544 14.019531 12.009766 12.350623 12.009766 10.390625 C 12.009766 8.7506266 11.050232 7.6701555 8.9902344 6.9101562 C 7.4602359 6.3401568 6.8203125 6.0097649 6.8203125 5.2597656 C 6.8203125 4.6697662 7.3895324 4.1308594 8.5195312 4.1308594 C 9.6495301 4.1308594 10.499219 4.4594534 10.949219 4.6894531 L 11.539062 2.5703125 C 10.839063 2.2403128 9.8905455 1.9804688 8.5605469 1.9804688 z "/>
+   /* Edited; Origin: https://gist.github.com/pabli24/85c406ebfdb840cb0450e029b28948b8 no idea what's with licensing, proably same as Stylus png icon which idk too. */
```

Comment:
That's a question for @Pabli24.

Reply:
If any image from the Stylus repo was used, that's under the GPL license so...